src/components/ContentNode/LinksBlock.vue

```diff
@@ -0,0 +1,40 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <TopicsLinkCardGrid v-if="!isListList" :identifiers="items" :topic-style="blockStyle" />
+</template>
+
+<script>
+import TopicsLinkCardGrid from 'docc-render/components/DocumentationTopic/TopicsLinkCardGrid.vue';
+import { TopicStyles } from 'docc-render/constants/TopicStyles';
+
+export default {
+  name: 'LinksBlock',
+  components: { TopicsLinkCardGrid },
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    },
+    blockStyle: {
+      type: String,
+      default: TopicStyles.compactGrid,
+    },
+  },
+  computed: {
+    isListList: ({ blockStyle }) => blockStyle === TopicStyles.list,
```

Comment:
I don't understand the name decision behind `isListList` 😄

Reply:
fixed